codegen-java/build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
   // apply the Pkl plugin
-  id("org.pkl-lang") version("0.30.0")
+    alias(libs.plugins.pkl)
```

Comment:
We don't use version catalogs on purpose. The idea is that you should be able to copy/paste the contents of an entire subdirectory and it should still work.